File: abc/FizzBuzz/Lambda.java
public class Lambda {

    public static String fizzBuzz(int n) {
        String word = new String();

        if (n % 3 == 0) {
            word += "Fizz";
        }
        if (n % 5 == 0) {
            word += "Buzz";
        }

        if (word.isEmpty()) {
            word = String.valueOf(n);
        }

        return word;
    }

    public static void main(String[] args) {
        for (int i = 1; i <= 100; i++) {
            System.out.println(fizzBuzz(i));
        }
    }
}

File: abc/FizzBuzz/Lambda.php
<?php
    function fizzBuzz(int $n): string {
        $word = "";

        if ($n % 3 == 0) {
            $word .= "Fizz";
        }
        if ($n % 5 == 0) {
            $word .= "Buzz";
        }

        if (strlen($word) == 0) {
            $word = (string) $n;
        }

        return $word;
    }

    if (count(debug_backtrace()) == 0) {
        for ($i = 1; $i <= 100; $i++) {
            printf("%s\n", fizzBuzz($i));
        }
    }
?>
